android/src/main/kotlin/com/mapbox/maps/mapbox_maps/MapboxOptionsController.kt

```diff
@@ -39,6 +42,10 @@ class MapboxOptionsController : _MapboxMapsOptions, _MapboxOptions {
     return ""
   }
 
+  override fun getFlutterAssetPath(flutterAssetUri: String?): String? {
+    return flutterAssetUri?.replace("""^asset://(.+)""".toRegex(), "asset://${flutterAssets.getAssetFilePathBySubpath("$1")}")
```

Comment:
Do we actually need regexp here? Can we replace a `asset://` prefix with the full path instead?

Reply:
> The returned file path is relative to the Android app's standard assets directory. Therefore, the returned path is appropriate to pass to Android's AssetManager, but the path is not appropriate to load as an absolute path.

Since the path is not an absolute path, I have to keep `asset` scheme here so core renderer would understand it

Reply:
I meant to replace `asset://` with `asset:///var/mobile/full/path` way.

Reply:
I do not know how to obtain the absolute path of the asset in Android 😅